app/src/androidTest/java/org/simple/clinic/di/TestAppComponent.kt

```diff
@@ -19,6 +19,7 @@ import org.simple.clinic.overdue.communication.CommunicationRepositoryAndroidTes
 import org.simple.clinic.overdue.communication.CommunicationSyncAndroidTest
```

Comment:
Let's rename this commit to just `Add method to fetch drugs for a protocol`. It's implied that the work will also be accompanied with automated tests.